src/public/app/widgets/dialogs/about.js

```diff
@@ -8,11 +8,8 @@ const TPL = `
     <div class="modal-dialog modal-lg" role="document">
         <div class="modal-content">
             <div class="modal-header">
-                <h5 class="modal-title mr-auto">${t("about.title")}</h5>
-
-                <button type="button" class="close" data-dismiss="modal" aria-label="Close" style="margin-left: 0;">
-                    <span aria-hidden="true">&times;</span>
-                </button>
+                <h5 class="modal-title">${t("about.title")}</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
Background and maybe link color regression on dark mode:

<img width="794" alt="image" src="https://github.com/user-attachments/assets/abc3d6df-9886-4504-97eb-8322fd13aa7b">